pennylane_lightning/core/src/simulators/lightning_qubit/gates/AssignKernelMap_Default.cpp

```diff
@@ -384,4 +384,19 @@ void assignKernelsForControlledMatrixOp_Default() {
                                all_threading, all_memory_model,
                                all_qubit_numbers, KernelType::LM);
 }
+void assignKernelsForSparseMatrixOp_Default() {
```

Comment:
If we could merge this with the existing `assignKernelsForMatrixOp_Default` instead to reduce the code duplication for sparse kernels?